pkg/apis/kops/v1alpha1/networking.go

```diff
@@ -25,6 +25,7 @@ type NetworkingSpec struct {
 	Kopeio   *KopeioNetworkingSpec   `json:"kopeio,omitempty"`
 	Weave    *WeaveNetworkingSpec    `json:"weave,omitempty"`
 	Calico   *CalicoNetworkingSpec   `json:"calico,omitempty"`
+	Canal    *CanalNetworkingSpec    `json:"canal,omitempty"`
```

Comment:
A note for you - you are going to need to generate the API machinery for this as well. See https://github.com/kubernetes/kops/blob/master/docs/development/api_updates.md